.dm-hooks/applypatch-msg.sample

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
I feel like we probably don't need all the sample files.

Reply:
removed!